src/webots/gui/WbVideoRecorderDialog.cpp

```diff
@@ -99,14 +99,15 @@ WbVideoRecorderDialog::WbVideoRecorderDialog(QWidget *parent, const QSize &curre
   mainLayout->setSizeConstraint(QLayout::SetFixedSize);
 
   // set default values
-  int quality, resolutionIndex;
-  double acceleration;
-  bool caption;
-  WbPreferences::instance()->moviePreferences(resolutionIndex, quality, acceleration, caption);
-  mResolutionList.setCurrentRow(resolutionIndex);
-  mQualitySpinBox->setValue(quality);
-  mAccelerationSpinBox->setValue(acceleration);
-  mCaptionCheckBox->setChecked(caption);
+  int qualityPreference, resolutionIndexPreference;
+  double accelerationPreference;
+  bool captionPreference;
```

Comment:
These variable names doesn't seem very well chosen.
They are way longer than before without adding any additional information.

In general I  have the feeling that the code became worse and less clear after resolving the shadow variable cppcheck error rather than improving the quality and readability.

Reply:
Personally, I think that the new variable names chosen by @ygoumaz are a better option in this particular case.

Other abbreviations that were used but didn't really improve the readability of the code where for examples `sc` meaning solid children, `tb` meaning toolbar, etc. 
If the definition is very close to the instruction it uses it, then it is fine, but this is not always the case in this PR.

Reply:
I think I agree with @stefaniapedrazzi on the proximity of the declaration and where it is used. Abbreviations should be used if the declaration is close to where the variable is used. In other cases, I think it would be more convenient to have a longer variable name (even a little redundant), like `name` becomes `nameString` or `nameArray`.